README.md

```diff
@@ -124,3 +124,8 @@ Then mount it as a volume using `-v /path/to/mydir:/etc/docsrv` and `-v /path/to
 2. `/` without any version has the second highest predecence and acts as if it was `/latest/`.
 3. `/$VERSION/$PATH` has the lowest precedence.
 4. `/var/www/public/errors/$PATH`
+
+### Develop
+
+You may use the `dotenv.example` file as a template for a `.env` file. There
```

Comment:
Usually is in the developer configuration. A .env file loads some env variables when in a dir (several plugins and tools for this). Useful way to have them per project and not global.